projects/ades1754/src/examples/def_uart/def_uart_example.c

```diff
@@ -0,0 +1,104 @@
+/***************************************************************************//**
+ *   @file   def_uart_example.c
```

Comment:
i don't really like the `default` name, you probably need better names for the examples